src/DateFormat/Language.elm

```diff
@@ -494,3 +494,134 @@ toPortugueseWeekdayName weekday =
 
         Sun ->
             "Domingo"
+
+
+
+-- French
+
+
+{-| The French language!
+-}
+french : Language
+french =
+    Language
+        toFrenchMonthName
+        toFrenchMonthAbbreviation
+        toFrenchWeekdayName
+        (toFrenchWeekdayName >> String.left 3)
+        toEnglishAmPm
+        toFrenchOrdinalSuffix
+
+
+toFrenchMonthName : Month -> String
+toFrenchMonthName month =
+    case month of
+        Jan ->
+            "janvier"
+
+        Feb ->
+            "février"
+
+        Mar ->
+            "mars"
+
+        Apr ->
+            "avril"
+
+        May ->
+            "mai"
+
+        Jun ->
+            "juin"
+
+        Jul ->
+            "juillet"
+
+        Aug ->
+            "août"
+
+        Sep ->
+            "septembre"
+
+        Oct ->
+            "octobre"
+
+        Nov ->
+            "novembre"
+
+        Dec ->
+            "décembre"
+
+
+toFrenchMonthAbbreviation : Month -> String
+toFrenchMonthAbbreviation month =
+    case month of
+        Jan ->
+            "janv"
+
+        Feb ->
+            "févr"
+
+        Mar ->
+            "mars"
+
+        Apr ->
+            "avr"
+
+        May ->
+            "mai"
+
+        Jun ->
+            "juin"
+
+        Jul ->
+            "juil"
+
+        Aug ->
+            "août"
+
+        Sep ->
+            "septembre"
```

Comment:
Is this a mistake or is *septembre* not abbreviated?

Reply:
That is of course a mistake.
And here I thought I had been extra careful... Oh well.

Good catch, thank you!